std_msgs/Header header
float64 co2_mass            # Mass of CO2 in grams
float64 moisture_content    # Moisture content in percentage
float64 contaminants        # Level of contaminants in percentage
float64 temperature         # Temperature in Celsius
float64 dew_point           # Dew Point in Celsius
